armometadata/k8sutils.go

```diff
@@ -102,3 +104,45 @@ func LoadConfig(configPath string) (*ClusterConfig, error) {
 	err = json.Unmarshal(res, &config)
 	return config, err
 }
+
+// ExtractMetadataFromBytes extracts metadata from the JSON bytes of a Kubernetes object
+func ExtractMetadataFromJsonBytes(input []byte) (error, map[string]string, map[string]string, string) {
```

Comment:
@matthyx we need also ownerReferences and resourceVersion

Reply:
done